docs/docs/micromatch-wrapper.md

```diff
@@ -0,0 +1,52 @@
+#### Overview
+[Micromatch](https://github.com/micromatch/micromatch) support various matching features, such as Glob pattern, Advanced globbing.
+
+Micromatch is a nodejs application, and MBT is a go application. In order to enable MBT to use the functionalities provided by micromatch, from MBT v1.2.25, we create [micromath wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) to package it.
+
+#### Install
+You can install the Micromatch Wrapper using either of these methods below:
+
+ **Download and install manually**
+
+   - [Download](https://github.com/SAP/cloud-mta-build-tool/releases) the latest binary file according to your operating system.
+    
+   - Extract the archive file to the folder where you want to install the tool.
+
+   - Add the binary file to your `~/bin` path according to your operating system:  
+
+     * In Darwin / Linux, copy binary file `micromatch-wrapper` to the `~/usr/local/bin/` folder, for example: `cp micromatch-wrapper /usr/local/bin/`

```

Comment:
* In Darwin / Linux, copy the `micromatch-wrapper` binary file to the `~/usr/local/bin/` folder, for example: `cp micromatch-wrapper /usr/local/bin/`

Reply:
OK